kbcstorage/workspaces.py

```diff
@@ -166,7 +167,8 @@ def load_files(self, workspace_id, file_mapping, preserve=None):
         file_list = files.list(tags=file_mapping['tags'])
         inputs = {}
         for file in file_list:
-            inputs[file['id']] = file_mapping['destination']
+            # append the file name and id to the path because destinations must be unique
+            inputs[file['id']] = file_mapping['destination'] + '/' + file['name'] + '/' + str(file['id'])
```

Comment:
@odinuv unfortunately I don't think so, because you can't have multiple inputs here with the same destination (which means 2 files with the same name)